tt_metal/impl/kernels/data_types.hpp

```diff
@@ -21,6 +21,7 @@ enum NOC : uint8_t {
 enum Eth : uint8_t {
```

Comment:
I removed this from main, because there was never a use case for SENDER vs RECEIVER. Maybe we can put `eth_mode` back and have a default value for all user kernels.

Reply:
Right, I had to add it back to differentiate dispatch on ethernet kernel from other user kernels.
SENDER, RECEIVER may be obsolete with your refactoring of tunneling, so I'll probably change that so some other generic name.